database/migrations/2018_06_04_129400_create_message_event_definition_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateMessageEventDefinitionTable extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('MESSAGE_EVENT_DEFINITION', function (Blueprint $table) {
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.